Niclas Jensen - Happy Thoughts

jnbjensen opened the pull request:
Link to my deployed project: 

https://jnbjensen-happy-thoughts.netlify.app/

My reflections on how this week's project turned out:

Still catching up on React after taking a 2-week hiatus from Technigo, so I found this project very challenging. I'm happy with the end result, but I've relied a lot on peeking at other people's code. My focus was on ensuring that I properly understood how the app works, and I did so by adding thorough explanations for every block of code via comments.

Things I'd like to have clarified or explained in more detail:

I'm still struggling with APIs and React props in particular; I'll be spending more time on those two topics in the days to come.

jnbjensen committed:
Initial commit with test component

jnbjensen committed:
Set up components and structure

jnbjensen committed:
Functioning app, zero CSS

jnbjensen committed:
Reconfiguring code

jnbjensen committed:
Fully commented code, only styling left

jnbjensen committed:
Styling added

jnbjensen committed:
Added text to Readme and PR template

jnbjensen committed:
Filled in the correct Readme this time

hannanorgard reviewed (commented):
Hi Nick! As usual, super great job this week! Your commenting is always super clear and helpful, especially when I think I grasped something, I read your comments and they make me reflect a bit extra. Your website looks super clean, very nice design! Keep up the good work! :D

jnbjensen committed:
Updated API url

jnbjensen committed:
Fix fetch request error

Changed the fetch method in handleNewLikeSubmit from
POST to PATCH. Error occurred due to the old API
having a POST endpoint for submits.

Also made some styling changes.

jnbjensen committed:
Updated title in html file

jnbjensen committed:
Change API URL (migrating from Google Cloud to Render)

jnbjensen committed:
Back to Google Cloud
